packages/ui/src/components/va-counter/VaCounter.vue

```diff
@@ -137,6 +138,8 @@ const { createEmits: createFieldEmits, createListeners: createFieldListeners } =
   { listen: 'click-append-inner', emit: 'click:increase-icon' },
 ])
```

Comment:
![image](https://github.com/epicmaxco/vuestic-ui/assets/35342240/b2759009-8711-4e93-8792-dc61103f6965)
Make width of default counter 120px